### PR TITLE
Update TTS task content

### PR DESCRIPTION
**Pull request by Vaibhavs10**

### CODE REVIEW

**Comment by Vaibhavs10** [comment]
FYI! This PR is to mainly introduce the TTS pipeline newly introduced in transformers. 
The Inference API bits aren't working atm and return a `overloaded` or `time out`
I'm working with Nico to fix that seperately.

File: tasks/src/text-to-speech/about.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ## Use Cases
 
-Text-to-Speech (TTS) models can be used in any speech-enabled application that requires converting text to speech.
+Text-to-Speech (TTS) models can be used in any speech-enabled application that requires converting text to speech imitatting human voice.
```
Comment (ylacombe) [inline tasks/src/text-to-speech/about.md:3]:
```suggestion
Text-to-Speech (TTS) models can be used in any speech-enabled application that requires converting text to speech imitating human voice.
```

**Review by ylacombe — APPROVED** [review]
LGTM providing that the Inference API works !

File: tasks/src/text-to-speech/about.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
 ## Use Cases
 
-Text-to-Speech (TTS) models can be used in any speech-enabled application that requires converting text to speech.
+Text-to-Speech (TTS) models can be used in any speech-enabled application that requires converting text to speech imitating human voice.
```
Comment (merveenoyan) [inline tasks/src/text-to-speech/about.md:3]:
```suggestion
Text-to-Speech (TTS) models can be used in any speech-enabled application that requires converting text to human speech in audio form.
```

**Review by merveenoyan — APPROVED** [review]
Looks good to me! feel free to dismiss if the definition doesn't sound right

File: tasks/src/text-to-speech/about.md (hunk 2 above)
Comment (Vaibhavs10) [reply tasks/src/text-to-speech/about.md:3]: I just thought about this. This sounds a bit weird. Is it okay if we can ignore this? :)

**Comment by Vaibhavs10** [comment]
Going to merge now, and can follow-up with a PR later on.

File: tasks/src/text-to-speech/about.md [hunk 3]
```diff
@@ -12,30 +12,32 @@ TTS models are widely used in airport and public transportation announcement sys
...
 def query(payload):
 	response = requests.post(API_URL, headers=headers, json=payload)
 	return response
 
-output = query({"inputs": "This is a test"})
+output = query({"text_inputs": "This is a test"})
 ```
 
-You can also use libraries such as [espnet](https://huggingface.co/models?library=espnet&pipeline_tag=automatic-speech-recognition&sort=downloads) if you want to handle the Inference directly.
+You can also use libraries such as [espnet](https://huggingface.co/models?library=espnet&pipeline_tag=text-to-speech&sort=downloads) if you want to handle the Inference directly.
```
Comment (sanchit-gandhi) [inline tasks/src/text-to-speech/about.md:31]:
```suggestion
You can also use libraries such as [espnet](https://huggingface.co/models?library=espnet&pipeline_tag=text-to-speech&sort=downloads) or [transformers](https://huggingface.co/models?pipeline_tag=text-to-speech&library=transformers&sort=trending) if you want to handle the Inference directly.
```

File: tasks/src/text-to-speech/about.md [hunk 4]
```diff
@@ -12,30 +12,32 @@ TTS models are widely used in airport and public transportation announcement sys
 
 ## Inference
```
Comment (sanchit-gandhi) [inline tasks/src/text-to-speech/about.md:13]:
```suggestion
## Inference API
```

File: tasks/src/text-to-speech/about.md [hunk 5]
```diff
@@ -12,30 +12,32 @@ TTS models are widely used in airport and public transportation announcement sys
...
 	response = requests.post(API_URL, headers=headers, json=payload)
 	return response
 
-output = query({"inputs": "This is a test"})
+output = query({"text_inputs": "This is a test"})
 ```
 
-You can also use libraries such as [espnet](https://huggingface.co/models?library=espnet&pipeline_tag=automatic-speech-recognition&sort=downloads) if you want to handle the Inference directly.
+You can also use libraries such as [espnet](https://huggingface.co/models?library=espnet&pipeline_tag=text-to-speech&sort=downloads) if you want to handle the Inference directly.
+
```
Comment (sanchit-gandhi) [inline tasks/src/text-to-speech/about.md:32]:
```suggestion
## Direct Inference
```

**Review by sanchit-gandhi — COMMENTED** [review]
Thanks for updating this! As a follow-up, could we add `transformers` as one of the supported libraries for TTS?

**Pull request merged**